### PR TITLE
Fix predicate package

### PR DESCRIPTION
**Pull request by Ambrevar**

### CODE REVIEW

File: source/defclass-star.lisp [hunk 1]
```diff
@@ -253,29 +265,40 @@ Use the slot name directly:
...
-                                 *symbols-to-export*)))
-                           ,(package-name *package*)))
-                 (find-class ',name nil))
-              result))))))
+          `(progn
+             ,@(when *predicate-name-transformer*
+                 (let ((pred-name (funcall *predicate-name-transformer* name)))
+                   `((defun ,pred-name (object)
+                       (subtypep (type-of object) ',name))
+                     ,@(when *export-predicate-name-p*
```
Comment (attila-lendvai) [inline source/defclass-star.lisp:282]: this commit is repeated, it's also present in another PR.

see my comments there (don't use subtypep where typep is sufficient)

**Review by attila-lendvai — CHANGES_REQUESTED** [review]
please close one of the two PR's that contain the "generate predicate" commit.

**Comment by Ambrevar** [comment]
Sorry for the mess with the pull requests, I had mistakenly assumed that you had merged #6.

I've rebased and force-pushed here: this patchset contains both #6 and the fix.

File: source/defclass-star.lisp (hunk 1 above)
Comment (Ambrevar) [reply source/defclass-star.lisp:282]: Oh!  I just understood the difference between `typep` and `subtypep`!  OK, will fix just now.

Comment (Ambrevar) [reply source/defclass-star.lisp:282]: Done!

**Pull request merged**